### ISSUE TITLE
The default config  access logs log format not valid

### ISSUE THREAD

**Issue by netroby** [mosn/mosn#1013]
error msg:
```
2020-03-03 12:32:27,607 [FATAL] [mosn] [NewMosn] AddListener error:initialize listener access logger ./logs/access_ingress.log failed: undefined variable, name: StartTime

```

The default config section not valid:

```
 "access_logs": [
            {
              "log_path": "/home/admin/mosn/logs/access_ingress.log",
              "log_format": "%StartTime% %RequestReceivedDuration% %ResponseReceivedDuration% %REQ.requestid% %REQ.cmdcode% %RESP.requestid% %RESP.service%"
            }
          ]
```

**Comment by champly** [comment]
You can modify with this [config](https://github.com/mosn/mosn/blob/cc3a0d9114f5a3465c249ae0a0642258008cf3bc/pkg/types/accesslog.go#L21)
``` golang
%start_time% %request_received_duration% %response_received_duration% %bytes_sent% %bytes_received% %protocol% %response_code% %duration% %response_flag% %response_code% %upstream_local_address% %downstream_local_address% %downstream_remote_address% %upstream_host%
```

**Comment by nejisama** [comment]
this problems has been fixed in #995

**Comment by netroby** [comment]
not really. i was on the latest master branch

**Comment by nejisama** [comment]
fixed in #995, but the pr is not merged yet.
